superset/cli.py

```diff
@@ -530,7 +530,7 @@ def import_datasources(path: str, sync: str, recursive: bool) -> None:
             files.extend(path_object.rglob("*.yml"))
         contents = {path.name: open(path).read() for path in files}
         try:
-            ImportDatasetsCommand(contents, sync_columns, sync_metrics).run()
+            ImportDatasetsCommand(contents, sync_columns, sync_metrics, kwargs=sync_array).run()
```

Comment:
Would an alternative here just be to pass sync_columns and sync_metrics as keyword arguments? This way, no need to change the functionality of ImportDatasetsCommand. 
```suggestion
            ImportDatasetsCommand(contents, sync_columns=sync_columns, sync_metrics=sync_metrics).run()
```

Reply:
I could probably just remove sync_columns and sync_metrics as arguments because they don't even get used in the ImportDatasetsCommand class.

Reply:
Also, there seem to be unit tests for import_datasources, but specifically for the --sync argument. I'll have to make some for that.